[27-Apr-2022 07:24:03 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:24:03 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:24:34 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:24:34 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:24:38 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:24:38 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:25:16 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:25:16 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:25:36 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:25:36 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:28:48 UTC] PHP Fatal error:  Declaration of App\Exceptions\Handler::report(Exception $exception) must be compatible with Illuminate\Foundation\Exceptions\Handler::report(Throwable $e) in /home/chapter10asasa/public_html/app/Exceptions/Handler.php on line 41
[27-Apr-2022 07:28:48 UTC] PHP Fatal error:  Uncaught ReflectionException: Class App\Exceptions\Handler does not exist in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php:877
Stack trace:
#0 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#1 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#2 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#3 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#4 /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\Container\Conta in /home/chapter10asasa/public_html/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879
[27-Apr-2022 07:34:50 UTC] PHP Warning:  Module "mongodb" is already loaded in Unknown on line 0
[27-Apr-2022 07:35:02 UTC] PHP Warning:  Module "mongodb" is already loaded in Unknown on line 0
[27-Apr-2022 07:35:32 UTC] PHP Warning:  Module "mongodb" is already loaded in Unknown on line 0
